app/src/main/java/com/beemdevelopment/aegis/ui/fragments/preferences/ImportExportPreferencesFragment.java

```diff
@@ -483,7 +505,8 @@ public void onException(Exception e) {
         }
     }
 
-    private void onExportResult(int requestCode, int resultCode, Intent data) {
+    private void onExportResult(int requestCode, int resultCode, @Nullable Intent data) {
+        if (data == null) return;
```

Comment:
We don't shorten if statements like this. Instead:

```java
if (data == null) {
    return;
}
```

Please apply this to the rest of your changes as well.